igvc2023/config/model1.yaml

```
@@ -0,0 +1,30 @@
+# for wheel and imu fusion
+
+frequency: 30
+sensor_timeout: 0.1
+two_d_mode: true
+transform_time_offset: 0.0
+transform_timeout: 0.0
+print_diagnostics: true
+publish_tf: true
+publish_acceleration: false
+
+map_frame: map
+odom_frame: odom
+base_link_frame: base_link
```

機体のベースとなるTFフレームは`base_footprint`です。
```suggestion
base_link_frame: base_footprint
```